### ISSUE TITLE
BrowserWindowProxy, parameter offset in postMessage / addEvent Listener  

### ISSUE THREAD

**Issue by coclav** [atom/electron#3508]
`postMessage` to a child window  `BrowserWindowProxy` doesn't get the parameters accross

in my `parent` window I have 
```
// opens child page
var printWindow = window.open("template_printPreview.html");

// listens the call of the child window to send content
window.addEventListener(
    'message',
    function (event) {
        if ( event.data === "ready") {
            event.source.postMessage("my Content to print", "*")
        }
    },
    false);
```

on my `child` page:

```
// listens message event (from the parent)
window.addEventListener('message', function (event) {
    
    // do something with event.data
    // problem: event.data is always "*" and can't seem to retrieve "my content to print"

}, false);

// tells parent when ready
window.opener.postMessage('ready', '*');
```

**Comment by etiktin** [comment]
I just tried it with v0.35.1 on Windows and `event.data` contained the correct value ("my Content to print").
@coclav can you test it again with the latest version?

**Comment by coclav** [comment]
I confirm fixed on v0.35.0 too
